lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -456,20 +480,36 @@ def delete_comment(request, course_id, comment_id):
     return JsonResponse(prepare_content(comment.to_dict(), course_key))
 
 
+def _vote_or_unvote(request, course_id, obj, value):
+    """
+    Vote or unvote for a thread or a response.
+    """
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
```

Comment:
@sarina I've posted #10660 to clean this up.  It can only be merged after this PR, since I had to base it on the topic branch for this PR.  If there are further changes necessary to this PR, they need to be forward-ported.